cloudinit/patcher.py

```diff
@@ -39,10 +38,6 @@ def handleError(self, record):
 
 
 def patch():
-    imp.acquire_lock()
-    try:
-        _patch_logging()
-    finally:
-        imp.release_lock()
+    _patch_logging()
```

Comment:
The only place `patch` is called is in `main.py`. Can we remove the `patch` function altogether and rename `_patch_logging` to `patch_logging` and call that instead?

Reply:
Well I guess `patch()` was meant to potentially patch more things via separate `_patch_something()` calls, but that code was last touched in 2016 and I don't see what else we may need to patch that way, so +1 for this.